src/ec/packages/ec-component-link/__snapshots__/link.test.js.snap

```diff
@@ -31,7 +40,7 @@ exports[`EC - Link With icon after renders correctly 1`] = `
    
   <svg
     aria-hidden="true"
-    class="ecl-icon ecl-icon--fluid ecl-link__icon"
+    class="ecl-icon ecl-icon--fluid ecl-link__icon ecl-test-extra-class"
```

Comment:
there shouldn't be an extra class here I guess, as it is not in the test description

Reply:
why not? it looks like it is defined at line 209 of link.test.js
`icon: {
        type: 'ui',
        name: 'external',
        size: 'fluid',
        path: defaultIconPath,
        extra_classes: 'ecl-test-extra-class',
      }
`

Reply:
Ok, I thought that it was a parameter leak